app/views/manage/reports/show.html.haml

```diff
@@ -0,0 +1,12 @@
+%h3= params[:action]
+%div(style="padding-bottom: 20px;")= @description
```

Comment:
Most of the HAML seems to use things like `div.mb-2` or `div.mb-4` for this. It would be good to try and match that style, I think.